packages/sync-server/src/load-config.js

```
@@ -88,6 +88,7 @@
   projectRoot,
   multiuser: false,
   token_expiration: 'never',
+  userCreationMode: 'manual'
```

ESLint

[error] Insert `,`

(prettier/prettier)